suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
Could we rename this file to `tier-2_cephfs_test-volume-management.yaml`?

Reply:
tier-2 folder is not required ??

Reply:
Ideally, it is not required but there is not harm in having it a separate folder.